### PR TITLE
[Static Reference] Lps 170398 account

### PR DESCRIPTION
**Pull request by slnn**
https://issues.liferay.com/browse/LPS-170398

### CODE REVIEW

File: modules/apps/account/account-admin-web/src/main/java/com/liferay/account/admin/web/internal/helper/CurrentAccountEntryManagerHelper.java [hunk 1]
```diff
@@ -0,0 +1,39 @@
...
+
+import org.osgi.service.component.annotations.Activate;
+import org.osgi.service.component.annotations.Component;
+import org.osgi.service.component.annotations.Reference;
+
+/**
+ * @author Lily Chi
+ */
+@Component(service = {})
+public class CurrentAccountEntryManagerHelper {
```
Comment (dantewang) [inline modules/apps/account/account-admin-web/src/main/java/com/liferay/account/admin/web/internal/helper/CurrentAccountEntryManagerHelper.java:28]: @slnn I think you can combine the helper components you added in this commit and later commits into one.

**Comment by zhangyan0701** [comment]
Semantic Versioning error:
```
     [exec] > Task :apps:account:account-api:baseline
     [exec]   PACKAGE_NAME                                       DELTA      CUR_VER    BASE_VER   REC_VER    WARNINGS  
     [exec] = ================================================== ========== ========== ========== ========== ==========
     [exec]   com.liferay.account.manager                        UNCHANGED  2.0.0      1.0.0      1.0.0      EXCESSIVE VERSION INCREASE
     [exec] 	-   version    1.0.0
     [exec] 	+   version    2.0.0
     [exec] [Baseline Report] Mode: diff (persisted)
     [exec] Semantic versioning is incorrect while checking /opt/dev/projects/github/liferay-portal/tools/sdk/dist/com.liferay.account.api-14.0.0.jar against /opt/dev/projects/github/liferay-portal/.gradle/caches/modules-2/files-2.1/com.liferay/com.liferay.account.api/13.0.0/8823ba07f04cf873fc01f3f75b9dfc5d7312a940/com.liferay.account.api-13.0.0.jar
```
Full console log:
https://testray.liferay.com/reports/production/logs/2022-12/test-1-32/test-portal-acceptance-pullrequest(master)/7278/modules-semantic-versioning-jdk8/0/0/jenkins-console.txt.gz?authuser=0

Other failures are unrelated.

**Comment by slnn** [comment]
ci:test:relevant

**Comment by slnn** [comment]
ci:test:sf

**Comment by dantewang** [comment]
@slnn FYI I changed your PR title. Please make the title format consistent with others when you send more.

**Comment by dantewang** [comment]
See https://github.com/liferay-core-infra/liferay-portal/pull/2670

**Pull request closed without merging**